src/math/mod.rs

```diff
@@ -1,5 +1,6 @@
 mod armstrong_number;
 mod baby_step_giant_step;
+mod derrivative_method;
```

Comment:
```suggestion
mod derivative_method;
```

Note that the file + several occurrences of derivative have this typo.